Model/Resolver/Shipments.php

```diff
@@ -24,11 +24,11 @@ class Shipments extends BaseShipments
      */
     public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
     {
-        if (!isset($value['model']) && !($value['model'] instanceof Order)) {
+        if (!isset($value['model']) && !($value['model'] instanceof OrderInterface)) {
```

Comment:
Confused. In core magento modules it is checking for invoices instanceof OrderInterface, for shipments it was checking just instanceof Order :D

Reply:
I'd say give it a try and see if it breaks
Interface is safer